### DISCUSSION TITLE
Can't find the snippet view panel.

### DISCUSSION THREAD

**Discussion by LBR-13** [James-Yu/LaTeX-Workshop#3168] [Q&A] [answered]
![R3653K%KWW)ZH4MZ4%@DM48](https://user-images.githubusercontent.com/78533214/155871901-f0aebf93-c90a-43ea-a704-a0053e3f8cff.png)

**Comment by jlelong** [comment]
Please post the complete logs of the extension starting from `Initializing LaTeX Workshop`. Do not truncate or alter them .

![74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524](https://user-images.githubusercontent.com/2910140/77959270-98c77e00-72d6-11ea-8c58-4934a9fde39a.png)

**Comment by LBR-13** [comment, marked as answer]
I have solved this problem, thank you.
